app/helpers/application_helper/toolbar/auth_key_pair_cloud_center.rb

```diff
@@ -12,6 +12,7 @@ class ApplicationHelper::Toolbar::AuthKeyPairCloudCenter < ApplicationHelper::To
           t = N_('Remove this Key Pair from Inventory'),
           t,
           :url_parms => "&refresh=y",
+          :klass     => ApplicationHelper::Button::AuthKeyPairCloudDelete,
```

Comment:
we should use the generic class with support feature mixin, like e.g. this https://github.com/Ladas/manageiq-ui-classic/blob/79bb60304b6526e9141e5365f8b72c9d5b518d9d/app/helpers/application_helper/toolbar/cloud_network_center.rb#L18